python_ta/z3/z3_parser.py

```diff
@@ -38,6 +38,9 @@ def parse(self, node: astroid.NodeNG) -> z3.ExprRef:
         Convert astroid node to z3 expression and return it.
         If an error is encountered or a case is not considered, return None.
         """
+        if isinstance(node, nodes.Call):
```

Comment:
Instead of creating a separate if statement, add this as an additional branch below.